### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by Johnmichael** [brotalnia/BlizzardPlates#1]
This addon is not loading on the addon selection menu.

**Comment by brotalnia** [comment]
Make sure the folder of the addon is named exactly "BlizzPlates" and not anything else.

**Comment by Johnmichael** [comment]
Of course. I didn't manage to get the one from github working, however I
found a reddit thread where you posted a version, and that one worked.
